course_catalog/management/commands/backpopulate_xpro_files.py

```diff
@@ -0,0 +1,38 @@
+"""Management command for populating xpro course run file data"""
+
+from django.core.management import BaseCommand
+
+from course_catalog.tasks import import_all_xpro_files
+from open_discussions.utils import now_in_utc
+
+
+class Command(BaseCommand):
+    """Populate xpro course run files"""
+
+    help = "Populate xpro course run files"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "-c",
+            "--chunk-size",
+            dest="chunk_size",
+            default=1000,
```

Comment:
This should be XPRO_ITERATOR_CHUNK_SIZE.

Reply:
The `OCW_ITERATOR_CHUNK_SIZE` value is only used in `get_ocw_data`. In the `backpopulate_ocw_files` management command it also uses a default of `1000`. @mbertrand any thoughts on whether I should use the chunk size here or leave it at `1000`?

Reply:
Since there is an option (`-c`) to change it to anything you want when running the command, I think it is okay to leave as is.  But `XPRO_ITERATOR_CHUNK_SIZE` isn't used anywhere so can be removed.

Reply:
Actually you can use that setting when scheduling the task